### DISCUSSION TITLE
Moving a movieclip/object inside of your Adobe animate scene

### DISCUSSION THREAD

**Discussion by havokentity** [bibixx/react-adobe-animate#55] [Q&A / Help] [answered]
Hola,

I was wondering if it was possible to move a MovieClip by drag and drop and perform a hitTest with another MovieClip instead of Adobe Animate with the logic in React?

Been fiddling around with getAnimationObject but didn't have much luck getting it to access the inside stuff in the Adobe Animate Scene.

Any help would be much appreciated.

Thanks!

**Comment by bibixx** [comment, marked as answer]
Hi!
To be honest I'm not too experienced with Adobe Animate's API. This package is only a wrapper around the config that is being output from Animate. What you get as an argument inside the `getAnimationObject` is the stage object. Can't really help much more I'm afraid 😞

**Reply by havokentity** [reply]
Thanks so much for the super fast response! I shall try to use what comes out of the `getAnimateObject` and hopefully get something working! Thank you once again!
